packages/internal-lib-build/typedoc.json

```
@@ -0,0 +1,9 @@
+{
+  "out": "../../docs",
+  "tsconfig": "../../tsconfig.json",
```

i see you added a tsconfig inside `internal-lib-build`, do you mean to use that one?

```suggestion
  "tsconfig": "./tsconfig.json",
```

---

The tsconfig in this lib is not working with the whole monorepo. I have to move it out to the root. 🤔